Tools/Scripts/rewrite-compile-commands

```diff
@@ -86,7 +86,8 @@ for entry in compile_commands:
                     })
                     break
             else:
-                print('Failed to find', include_path)
+                # TODO: Don't just ignore derived source that is not generated yet.
```

Comment:
Honestly, why not? Humans are not expected to edit these files, so is it really a big deal for them to be omitted?

Reply:
i.e. you can consider landing this without the TODO.

Reply:
I agree with you. Let's wait for @TingPing opinion.

Reply:
It is something I'd like to handle eventually but the TODO isn't super valuable. I have no strong opinion.

Reply:
Fixed.